app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(current_user.id)
```

Comment:
- [ ] All show pages for all users are currently only displaying the current user. -- we need find the user from the id params `users/1`

Reply:
Can you please create two users and two posts to see all the posts. Attached is the screenshot of the post index page
![Screen Shot 2020-01-09 at 11 02 04](https://user-images.githubusercontent.com/46388113/72053633-57b76300-32bf-11ea-97e4-f63c338ab156.png)